### ISSUE TITLE
Add alt tag for conference poster

### ISSUE THREAD

**Issue by G0dwin** [bikebike/BikeBike#141]
The conference poster really should have an alt tag, all it needs to say is `%{conference_title} poster` and in Spanish this should be `%{conference_title} póster`.
